src/pipelines/svm.py

```diff
@@ -30,27 +29,23 @@ def run_model(train_data, train_labels, test_data, test_labels, svm_C=1.0, svm_k
 
     # we create an instance of Neighbours Classifier and fit the data.
     svm.fit(train_data, train_labels)
-
-    #Now that we have something trained we can check if it is accurate with the test set
-    pred_labels = svm.predict(test_data)
-    perform_results = performance_metrics.get_perform_metrics(test_labels, pred_labels)
-
-    return pred_labels, perform_results
+    
+    return svm
 
 def main(argv):
 
-    train_data, train_target, test_data, test_target = argv[0], argv[1], argv[2], argv[3]
+    train_data, train_target = argv[0], argv[1]
 
-    if len(argv)>4:
-        args_dict = argv[4]
+    if len(argv)>2:
+        args_dict = argv[2]
     else:
         args_dict = {}
 
     print("running support vector machine...",file=sys.stderr)
 
-    predicted_labels, perform_results = run_model(train_data, train_target, test_data, test_target, **args_dict)
+    svm = run_model(train_data, train_target, **args_dict)
 
-    return predicted_labels, perform_results
+    return svm
 
 if __name__ == '__main__':
     if len(sys.argv) < 5:
```

Comment:
The usage section needs be updated to reflect that we no longer expect sys.argv to have a length of 5 and the print statement should remove test_data and test_labels